install.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+import re
+import os
+import subprocess
+import shutil
+import argparse
+
+
+CHART = './fpga.chart.py'
+CONF = './fpga.conf'
+
+
+def get_flavour():
+    flavour_line = subprocess.check_output('cat /etc/os-release | grep -E \'^NAME="\'', shell=True)
+    flavour = re.match(r'^NAME=\"(\w+)', flavour_line.decode('UTF-8'))
+
+    return flavour.group(1)
+
+
+def check_for_package(cmd):
+    try:
+        DEVNULL = open(os.devnull, 'w')
+        out = subprocess.check_output(cmd + ' | grep netdata', stderr=DEVNULL, shell=True)
```

Comment:
- IMO the better way to suppress output is to use `stderr=subprocess.PIPE` and just ignore the pipe.
- Always use a context manager to `open`, saves you the `close`.
- Prefer to not use `shell=True` but `shlex.split()` for the `cmd`.
- grep twice?